dev/lint-python

```diff
@@ -17,6 +17,47 @@
 # limitations under the License.
 #
 
+VIRTUAL_ENV_DIR="build/venv"
+if [[ ! -z $VIRTUAL_ENV ]]; then
+    echo "We are in the following virtualenv: $VIRTUAL_ENV. Leaving it..."
+    # Cannot call "deactivate" from a bash script, copy/paste the code from virtualenv 'activate'
```

Comment:
I am open to suggestion. If we are already inside a venv we can expect the user knows what it does and don't desactivate it.